src/app/jobs/components/job-detail/components/job-result-table/job-results-table.component.ts

```diff
@@ -208,6 +210,16 @@ export class JobResultTableComponent implements AfterViewInit, OnInit, OnDestroy
     return `https://www.metanetx.org/equa_info/${hp.replace('DM_', '')}`;
   }
 
+  showWarning(method: string, checked: boolean): void {
+    if (method === 'PathwayPredictor+DifferentialFVA' && !checked) {
+      this.snackBar.open(`Sharing of DifferentialFVA designs is not completely supported yet.
+      Proceed with caution if you want to inspect the predicted pathways
+      and knockouts on the interactive map.`, '', {
+        duration: 5000,
```

Comment:
I would increase the duration, to 10 or maybe even 15 seconds, as it's a bit of a mouthful to read and understand in just 5 seconds.

Also, I would maybe change `Sharing of` to `Visualizing`, since that is the label used on the button, so it's easier for the user to make the connection. I usually think of sharing as sending something to other people.